### ISSUE TITLE
Setting pageSize not parsing string

### ISSUE THREAD

**Issue by jaandric** [fooplugins/FooTable#588]
Added custom filter for pageSize, if we pass (selected) string as pageSize parameter to FooTable.get('#mainTable').pageSize(selected); it will not work and no errors will be thrown. Lost  2 hours on this

Full code:
```
jQuery('.footable').footable({
	components: {
		filtering: FooTable.Filtering.extend({
			construct: function(instance){
				this._super(instance);
				this.pageSizes = ['20','50','100','200'];
				this.def = '10';
				this.$pageSize = null;
			},
			$create: function(){
				this._super();
				var self = this,
					$form_grp = $('<div/>', {'class': 'form-group'})
						.append($('<label/>', {'class': 'sr-only', text: 'Page Size'}))
						.prependTo(self.$form);
				self.$pageSize = $('<select/>', { 'class': 'form-control'})
					.on('change', {self: self}, self._onPageSizeDropdownChanged)
					.append($('<option />', {text: self.def, value : self.def}))
					.appendTo($form_grp);

				$.each(self.pageSizes, function(i, pageSize){
					self.$pageSize.append($('<option/>',{ value : pageSize}).text(pageSize));
				});
			},
			_onPageSizeDropdownChanged: function(e){
			         var self = e.data.self,
					selected = $(this).val();
					FooTable.get('#mainTable').pageSize(parseInt(selected)); //parse to int
			},
		})
	}
});
```

**Comment by steveush** [comment]
Hi,

Sorry you lost two hours on this but it explicitly states in the code that a number is the required param type for that method or nothing at all. You can see this on [line 387 of the FooTable.Paging components source](https://github.com/fooplugins/FooTable/blob/V3/src/js/components/paging/FooTable.Paging.js#L387) which is then exposed in the [JsDocs](http://fooplugins.github.io/FooTable/docs/jsdocs/FooTable.Paging.html#pageSize).

Also the reason that no error is thrown is that if the parameter supplied is not a number the current page size is returned. The method performs a `GET` **or** a `SET` operation depending on the parameter.

That said I have pushed a [commit](https://github.com/fooplugins/FooTable/commit/8dcdc92dbbd15e1a779ea6b8e50d71c80c37d764) to the develop branch which changes the method to use `parseInt` internally allowing it to accept strings as it does make sense to allow them if valid. This will be available in the next version release or you can simply checkout and build the [develop branch](https://github.com/fooplugins/FooTable/tree/develop) or use the files found in the [compiled folder](https://github.com/fooplugins/FooTable/tree/develop/compiled).

Thanks

**Comment by steveush** [comment]
The above changes are now available in the 3.1.5 version.
